qemu/tests/hotplug_mem_share_discard_data.py

```diff
@@ -0,0 +1,132 @@
+import math
+
+from avocado.core import exceptions
+
+from virttest import error_context
+from virttest import env_process
+from virttest import test_setup
+
+from avocado.utils import memory
+from avocado.utils import process
+
+from virttest.utils_numeric import normalize_data_size
+from virttest.utils_test.qemu import MemoryHotplugTest
+from virttest.staging import utils_memory
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Memory share and discard-data hotplug test
+    1) Setup hugepages
+    2) Mount /mnt/kvm_hugepage
+    3) Boot guest with one numa node
+    4) Hotplug 1G memory
+    5) Check memory
+    6) Unplug memory
+    7) Check memory
+    8) Check results for discard-data value
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    # Deletes mem-path file to avoid test error
+    process.system("rm -rf %s" % params["mem-path_plug1"])
+    hugepage_size = memory.get_huge_page_size()
+    mem_size = int(normalize_data_size("%sM" % params["mem_fixed"], "K"))
+
+    node_num = params.get_numeric("node_num")
+    prealloc_mem = params.get("prealloc_mem", "yes")
+    mem_devs = ""
+    guest_numa_nodes = ""
```

Comment:
Overwrite this parameter in cfg L15 ?
```
    guest_numa_nodes = "node0"
```
Could you please explain the reason for this change, thanks!